src/main/java/spoon/reflect/factory/TypeFactory.java

```diff
@@ -282,7 +282,7 @@ public TypeFactory() {
 	}
 
 	/**
-	 * Creates a reference to a one-dimension array of given type.
```

Comment:
"a one" is correct, see https://www.quora.com/Which-is-correct-a-one-or-an-one

Reply:
@monperrus 
That's right. Thank you for your careful review!